pkg/operator/ceph/cluster/osd/status.go

```diff
@@ -185,7 +196,19 @@ func (c *Cluster) completeOSDsForAllNodes(config *provisionConfig, configOSDs bo
 				if !ok {
 					logger.Infof("orchestration status config map result channel closed, will restart watch.")
 					w.Stop()
-					<-time.After(100 * time.Millisecond)
+					<-time.After(5 * time.Second)
+					leftNodes := 0
+					leftRemaingNodes := util.NewSet()
+					leftNodes, leftRemaingNodes, completed, statuses, err = c.checkNodesCompleted(selector, config, configOSDs)
```

Comment:
do we need to worry about `statuses` getting overwritten upon error also? if an error is returned and `statuses` is overwritten, we've lost the resource version that gets set in the watch options on line 180.  is that a problem?

Reply:
That'll be the same error case as when the cm are listed the first time.